packages/atomic/src/locales.json

```diff
@@ -612,6 +612,14 @@
     "zh-CN": "没有找到与 {{query}} 相关的结果",
     "zh-TW": "沒有找到與 {{query}} 相關的結果"
   },
+  "showing-results-for": {
+    "en": "Showing results for {{query}}",
+    "fr": "Résultats affichés pour {{query}}"
+  },
+  "search-instead-for": {
+    "en": "Search instead for {{query}}",
```

Comment:
```suggestion
    "en": "Search for {{query}} instead",
```

Reply:
Discussed it with Johnny, and visually it's better to keep the query at the end.

Reply:
Oh I see, no worries. This one works decently well both ways.